source/clear-linux/guides/deploy-at-scale.rst

```diff
@@ -144,39 +122,28 @@ other software development lifecycle. Below are some pointers on this subject:
     - Measure against metrics and indicators which are relevant to your 
       business (e.g. performance, power, etc) from release to release.
 
-    - Integrate with your organization's governance processes, such as change 
+    - Integrate with your organization's governance processes, such as change
       control.
 
-
-
-
-
-
-
 Versioning Infrastructure
 -------------------------
 
-|CL| version numbers have a deep meaning as they version of the whole 
-infrastructure stack  - from the OS components to libraries to applications. 
+|CL| version numbers are very important as they apply to the whole 
+infrastructure stack from OS components to libraries and
+applications. 
 
-Good record keeping can be powerful here.You should keep a detailed registry 
+Good record keeping is important, so you should keep a detailed registry 
```

Comment:
The use of the word important shortly after each other sounds a bit repetitive. I'd like to convey in the second sentence that good record keeping is not only important but also rewarding or enlightening. 

Can you provide an alternative phrasing?

Reply:
necessary?

Reply:
or critical?